src/templates/blog-post.tsx

```diff
@@ -8,6 +8,7 @@ import { Markdown } from '../components/markdown/markdown';
 import { SectionTitle } from '../components/typography/typography';
 import styled from 'styled-components';
 import { up } from '../components/breakpoint/breakpoint';
+import Signature from '../components/signature/signature';
```

Comment:
Maybe we can rename from "signature" to "byline" as it's the usual name for the author addition to an article.